WorkflowSwiftUIExperimental.podspec

```
@@ -18,8 +18,8 @@ Pod::Spec.new do |s|
 
     s.source_files = 'WorkflowSwiftUIExperimental/Sources/*.swift'
 
-    s.dependency 'Workflow', WORKFLOW_VERSION
-    s.dependency 'WorkflowUI', WORKFLOW_VERSION
+    s.dependency 'Workflow', "~> #{WORKFLOW_MAJOR_VERSION}.0"
+    s.dependency 'WorkflowUI', "~> #{WORKFLOW_MAJOR_VERSION}.0"
```

A possible alternative is hardcoding the version bound like this, which could save you from having to touch this file at all when you are bumping the major version of the other modules. But then you'd still have to bump WorkflowSwiftUIExperimental in a separate change for Register's sake.

Also, I feel like there may have been some other reason I didn't do this 🤔 

```suggestion
    s.dependency 'Workflow', "~> 3.0"
    s.dependency 'WorkflowUI', "~> 3.0"
```